runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -634,6 +634,22 @@ static TR_YesNoMaybe isValue(TR::VPConstraint *constraint)
    return TR::Compiler->cls.isValueTypeClass(clazz) ? TR_yes : TR_no;
    }
 
+static bool owningMethodDoesNotContainStoreChecks(OMR::ValuePropagation *vp, TR::Node *node)
+   {
+   TR::ResolvedMethodSymbol *method = node->getSymbolReference()->getOwningMethodSymbol(vp->comp());
```

Comment:
The node's owning method is more unique in the sense that it corresponds to the bytecode that the node was generated for. The symbol reference can be shared across different methods in cases when we were able to prove that the relevant constant pool references (for a field or a static for example) was referring to the same entity in the two methods.

Reply:
Thanks for the clarification Vijay! 

@hzongaro since VT and TreeLowering are trying to mirror ILGen, should the first one be used to get the owning method that corresponds to the bytecode of the node?

`   TR::ResolvedMethodSymbol *method = comp->getOwningMethodSymbol(node->getOwningMethod());
`

Reply:
Thanks for catching that serious mistake, Annabelle @a7ehuo!  I had that version of the test in the changes that we used for all of our experiments, but I accidentally dropped it in pulling together the changes for this pull request.

Reply:
Fixed in commit 3ddc5afce0d07fe7ad70e44b6a840d490f85aa49